extensions/smallrye-openapi-common/deployment/src/main/java/io/quarkus/smallrye/openapi/common/deployment/SmallRyeOpenApiConfig.java

```diff
@@ -50,6 +58,12 @@
     @ConfigItem(defaultValue = "Authentication")
     public String securitySchemeDescription;
 
+    /**
+     * This will automatically add the security requirement to all methods that has a `RolesAllowed` annotation.
```

Comment:
How about classes that are annotated with this?

Reply:
You mean what if the class already have the `@SecurityRequirement` annotation ? Then we just use the annotation value.

Reply:
No, I mean if the class is annotated with `@RolesAllowed`. Is that currently handled? Because reading this Javadoc I would guess that it isn't

Reply:
Ok, so you can annotation a class with `RolesAllowed` and that applies to all methods ? I did not think of that ? Let me see if I can add that.

Reply:
This now also works for class level annotations